tests/suite-unit/PluginFlyvemdmTaskstatus.php

```diff
@@ -35,6 +35,40 @@
 
 class PluginFlyvemdmTaskstatus extends CommonTestCase {
 
+    private $dataField = [
+        'group'     => 'application',
+        'symbol'    => 'deployApp',
+        'type_data' => '',
+        'unicity'   => 0,
+     ];
+     /**
+      * @return array
+      */
+     private function createNewTaskInstance() {
+        $class = new \PluginFlyvemdmTask();
+        $task = $this->newTestedInstance($class);
+        return $task;
+     }
```

Comment:
This method is useless. Replace calls to this method with $task = $this->newTestedInstance();